diff --git a/dist/plugins/email-otp/index.d.mts b/dist/plugins/email-otp/index.d.mts
index cf5cb0e..66a9521 100644
--- a/dist/plugins/email-otp/index.d.mts
+++ b/dist/plugins/email-otp/index.d.mts
@@ -351,13 +351,13 @@ declare const emailOTP: (options: EmailOTPOptions) => {
           responses: {
             200: {
               description: string;
-              contnt: {
+              content: {
                 "application/json": {
                   schema: {
-                    type: string;
+                    type: "object";
                     properties: {
                       success: {
-                        type: string;
+                        type: "boolean";
                       };
                     };
                   };
diff --git a/dist/plugins/email-otp/routes.mjs b/dist/plugins/email-otp/routes.mjs
index af2e728..e87eb1d 100644
--- a/dist/plugins/email-otp/routes.mjs
+++ b/dist/plugins/email-otp/routes.mjs
@@ -579,7 +579,7 @@ const resetPasswordEmailOTP = (opts) => createAuthEndpoint("/email-otp/reset-pas
 		description: "Reset password with email and OTP",
 		responses: { 200: {
 			description: "Success",
-			contnt: { "application/json": { schema: {
+			content: { "application/json": { schema: {
 				type: "object",
 				properties: { success: { type: "boolean" } }
 			} } }
